src/cat_generator.js

```diff
@@ -0,0 +1,13 @@
+    const randomIndex = Math.floor(Math.random() * names.length);
+    return names[randomIndex];
```

Comment:
The edge case we discussed offline essentially translates to "is `names[names.length]` ever out of bounds for an array", so this should be good.